src/garage/torch/modules/multi_headed_mlp_module.py

```diff
@@ -61,55 +63,43 @@ def __init__(self,
 
         output_dims = self._check_hidden_layer_parameter(
             'output_dims', output_dims, n_heads)
-        output_nonlinearities = self._check_hidden_layer_parameter(
-            'output_nonlinearities', output_nonlinearities, n_heads)
         output_w_inits = self._check_hidden_layer_parameter(
             'output_w_inits', output_w_inits, n_heads)
         output_b_inits = self._check_hidden_layer_parameter(
             'output_b_inits', output_b_inits, n_heads)
 
-        prev_size = input_dim
-        for size in hidden_sizes:
-            layer = nn.Sequential()
-            if layer_normalization:
-                layer.add_module('layer_normalization',
-                                 nn.LayerNorm(prev_size))
-
-            linear_layer = nn.Linear(prev_size, size)
-            hidden_w_init(linear_layer.weight)
-            hidden_b_init(linear_layer.bias)
-            layer.add_module('linear', linear_layer)
+        self._hidden_nonlinearity = hidden_nonlinearity
```

Comment:
are these also constructor args but unimplemented in the other MLPModules? It's very misleading for a constructor to have a parameter which is unused.

Reply:
`self._hidden_nonlinearity` is used in forward function (line 112)